network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -236,43 +236,43 @@ class ParticipatePage2(PrimaryPage):
 
     content_panels = Page.content_panels + [
         MultiFieldPanel(
-          [
-            ImageChooserPanel('ctaHero'),
-            FieldPanel('ctaHeroHeader'),
-            FieldPanel('ctaHeroSubhead'),
-            FieldPanel('ctaButtonTitle'),
-            FieldPanel('ctaButtonURL'),
-          ],
-          heading="Primary CTA",
-          classname="collapsible"
+            [
```

Comment:
As per slack: yeah these changes should not have been necessary =D

Reply:
@Pomax is this a dealbreaker? do you want me to go back and change these lines back?